index.bs

```diff
@@ -216,9 +216,9 @@ An aggregatable trigger data is a [=struct=] with the following items:
 : <dfn>source keys</dfn>
 :: An [=ordered set=] of [=strings=].
 : <dfn>filters</dfn>
-:: A [=filter map=].
+:: A [=filter map=] or a [=list=] of [=filter map=].
```

Comment:
Instead of saying this is one map or a list of maps, let's just have a list of maps, which will end up having size 1 if only one is supplied. Same below.